elements/rh-badge/rh-badge.ts

```diff
@@ -28,12 +30,17 @@ export class RhBadge extends LitElement {
   /**
    * Denotes the state-of-affairs this badge represents
    */
-  @property({ reflect: true }) state?:
+  @property({ reflect: true }) state:
+    | 'danger'
+    | 'warning'
+    | 'caution'
+    | 'neutral'
     | 'info'
     | 'success'
-    | 'moderate'
-    | 'important'
-    | 'critical';
+    | 'moderate' // deprecated
+    | 'important' // deprecated
+    | 'critical' = // deprecated
```

Comment:
`critical` is not showing as deprecated in [the docs](https://deploy-preview-1912--red-hat-design-system.netlify.app/elements/badge/code/#rh-badge) (assuredly due to the `=`).

Resize your browser to ~1100px to make this point clear.

Reply:
@adamjohnson I could be wrong, but I believe that `=` is the assignment of `neutral` should it be `null`.

Reply:
@adamjohnson OH, I get it. I'll see if I can fix that.

Reply:
The rhs after the = is the initializer

Reply:
To please the docs parser, I think we're going to have to do this:

```ts
  /**
   * Denotes the state-of-affairs this badge represents
   */
  @property({ reflect: true }) state:
    | 'danger'
    | 'warning'
    | 'caution'
    | 'neutral'
    | 'success'
    | 'moderate' // deprecated
    | 'important' // deprecated
    | 'critical' // deprecated
    | 'info' =
      'neutral';
```

Reply:
Or move the inline comments to the jsdoc

Reply:
Resolved by [7defe3b](https://github.com/RedHat-UX/red-hat-design-system/pull/1912/commits/7defe3b102061992cfc2057884527676f8fba9b7).